javascripts/tagmanager.js

```diff
@@ -1635,6 +1635,15 @@
                 throwError: throwError,
                 Container: Container,
                 addContainer: function (containerConfig, templates) {
+                    var setDebugFlag = urlHelper.getQueryParameter('setDebugFlag');
+                    if (setDebugFlag) {
+                        if (setDebugFlag == 1) {
+                            document.cookie = 'mtmPreviewMode=mtmPreview' + containerConfig.idsite + '_' + containerConfig.id + '%3D1;SameSite=Lax';
```

Comment:
I wonder if we should limit the time here ? I think the standard preview cookie restricts it to one week.

Reply:
Yes missed this, added a 7 day expiry